File: rest/src/main/java/de/philgenstock/choremaster/chore/ChoreRepository.java
package de.philgenstock.choremaster.chore;

import java.util.List;
import org.springframework.data.repository.CrudRepository;

public interface ChoreRepository extends CrudRepository<ChoreEntity, Long> {

  List<ChoreEntity> findByHouseHoldId(Long houseHoldId);
}

File: choremaster-rest/rest/src/main/kotlin/de/philgenstock/choremaster/chore/ChoreRepository.kt
package de.philgenstock.choremaster.chore

import de.philgenstock.choremaster.household.HouseHoldEntity
import org.springframework.data.repository.CrudRepository

interface ChoreRepository : CrudRepository<ChoreEntity, Long> {
    fun findByHouseHoldEntity(houseHoldEntity: HouseHoldEntity): MutableList<ChoreEntity>
}
